src/pages/cli/graphql/authorization-rules.mdx

```diff
@@ -32,10 +32,34 @@ In the example above, everyone (`public`) can read every Todo but owner (authent
 
 To help you get started, there's a global authorization rule defined when you create a new GraphQL schema. For production environments, remove the global authorization rule and apply rules on each model instead. 
 
+<BlockSwitcher>
+
+<Block name="Amplify CLI">
+
 ```graphql
 input AMPLIFY { globalAuthRule: AuthRule = { allow: public } }
 ```
 
+</Block>
+<Block name="AWS CDK">
+
+```graphql
+input AMPLIFY { globalAuthRule: AuthRule = { allow: public } }
+```
+
+In your CDK construct, you'll need to enable this "sandbox mode" explicitly:
+
+```ts
+new AmplifyGraphqlApi(this, "MyNewApi", {
+  ...,
+  translationBehavior: {
+    sandboxModeEnabled: true
+  }
+});
+```
+
+</Block>
```

Comment:
```suggestion
<Block name="AWS CDK">

In your CDK construct, you'll need to enable this "sandbox mode" via an input parameter, rather than as part of the Graphql schema definition:

```ts
new AmplifyGraphqlApi(this, "MyNewApi", {
  ...,
  translationBehavior: {
    sandboxModeEnabled: true
  }
});
```

</Block>
```